examples/cctl/hello-python/readme.md

```diff
@@ -0,0 +1,99 @@
+## Example python Controller
+This example is taken from [metacontroller](https://metacontroller.app/guide/create/) doc. There is an issue in this example that leads to continuous reconciliations. When _'CompositeController'_ is used with the parent resource's status **undefined** as a sub-resource, then the reconciliation gets into an infinite loop. This happens since _'CompositeController'_ updates _'status.observedGeneration'_ with _'metadata.generation'_ as part of its reconciliation.
+
+Following sums up the reconciliation logic when status is defined as a sub resource & vice-versa:
+
+1. If parent's status is defined as a sub resource then reconciliation logic patches this status using status api end point. This does not change resource's `metadata.generation` field.
+2. If parent's status is not defined as a sub resource then reconciliation logic updates the full parent object. This in turn leads to a update in parent's `metadata.generation` field. This leads to re-triggering of sync hook. In other words this forces the reconciliation to get into a never ending loop.
+
+**Solution**: Add below to watch's CRD
+```yaml
+  subresources:
+    status: {}
+```
+
+
+What changes you need to do? 
+
+You need to add below in your crd
+```yaml
+  subresources:
+    status: {}
+```
+### Use
+```yaml
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: helloworlds.example.com
+spec:
+  group: example.com
+  version: v1
+  names:
+    kind: HelloWorld
+    plural: helloworlds
+    singular: helloworld
+  scope: Namespaced
+  subresources:
+    status: {}
+```
+### Do not use
+```yaml
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: helloworlds.example.com
+spec:
+  group: example.com
+  version: v1
+  names:
+    kind: HelloWorld
+    plural: helloworlds
+    singular: helloworld
+  scope: Namespaced
+```
+
+### To try this example follow the below steps 
```

Comment:
@AmitKumarDas maybe we should align to one way of presenting examples, like using `kind` ? I know that it can be re-used for minikube etc, but it would be good to either :
* do not refer to platform (skipping the kind/docker/minikube/setup) and explaining how to set up in different section of documentation. Therefore every example (using only `kubectl` command) will be uniersal
* agree that all examples should based on assumption of using `kind` //`minikube`

Reply:
Actually created issue for that, https://github.com/AmitKumarDas/metac/issues/123 , so resolving the comment